packages/std/src/results/cosmos_msg.rs

```diff
@@ -19,14 +19,29 @@ where
     // to call into more app-specific code (whatever they define)
     Custom(T),
     Staking(StakingMsg),
+    Wasm(WasmMsg),
     /// A Stargate message encoded the same way as a protobof [Any](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto).
     /// This is the same structure as messages in `TxBody` from [ADR-020](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-020-protobuf-transaction-encoding.md)
     #[cfg(feature = "stargate")]
     Stargate {
         type_url: String,
         data: Binary,
     },
-    Wasm(WasmMsg),
+    /// Sends *native tokens* owned by the contract to the given address on another chain.
+    /// The channel must already be established between the ibctransfer module on this chain
+    /// and a matching module on the remote chain.
+    /// We cannot select the port_id, this is whatever the local chain has bound the ibctransfer
+    /// module to.
+    #[cfg(feature = "stargate")]
+    IbcSend {
```

Comment:
- Can we pull this out into a `IbcMsg` enum, which groups all IBC messages?
- Can we rename this to `Ics20Transfer` and link https://github.com/cosmos/ics/tree/master/spec/ics-020-fungible-token-transfer?

Reply:
Sure, I will rebase on another PR that adds IBC message types.

My issue was that the other ones (SendPacket and CloseChannel) only are valid for contracts that actively participate in ibc channels (and have 6 entry points for that). This one could be used by any contract.

I guess I can make that distinction with docstrings as well

Reply:
> My issue was that the other ones (SendPacket and CloseChannel) only are valid for contracts that actively participate in ibc channels (and have 6 entry points for that). This one could be used by any contract.
> 
> I guess I can make that distinction with docstrings as well

Yeah, I think this way would be better.

I guess this would be reflected in the necessary fiels as well (the other messages would require more fields that include all this channel state information)?